python/__init__.py

```diff
@@ -16,6 +16,33 @@ def get_version():
         return version_file.read().strip()
 
 
+def modcc():
+    import os
+    import sys
+    import subprocess
+
+    sys.exit(
+        subprocess.call(
```

Comment:
Does the pip version need work arounds? One can still use their own builds of these tools (because then you know where you've installed them).

The functions are there mainly to have something to bind a Python scripts to (see `[project.scripts]` in `pyproject.toml`). They're not meant to be used. We could prefix them with underscores to indicate that better?

Reply:
Something like this, or shove them into an `internal_111_donot_use` namespace.